### DISCUSSION TITLE
Please Stop Changing the Github UI!

### DISCUSSION THREAD

**Discussion by jbrucker** [community/community#67162] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

People use Github to develop applications, not to spend their time continually adjusting to Github's changing UI and page organization.

Many people, including me, prefer **stability** and **reliability** in their development tools. I don't like the clutter, the distractions, and the never-ending dialogs introducing minor new "features".  The pre-Microsoft Github was more efficient to use.

Github is showing the heavy hand of Microsoft, in my opinion.

**Comment by mr-cheffy** [comment]
How about some `opt out from new features`? People like I like to see new *features*.

**Comment by xieyuheng** [comment]
New product managers need to do something to github, otherwise ..., you know, they can not show they worded ...
